src/Entity/Jecoute/DataSurvey.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+namespace AppBundle\Entity\Jecoute;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use AppBundle\Entity\Adherent;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Table(name="jecoute_data_survey")
+ * @ORM\Entity
+
+ * @Algolia\Index(autoIndex=false)
+ */
+class DataSurvey
+{
+    /**
+     * @ORM\Column(type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\Adherent")
+     * @ORM\JoinColumn(onDelete="SET NULL")
+     */
+    private $author;
+
+    /**
+     * @var \DateTime
+     *
+     * @ORM\Column(type="datetime")
+     *
+     * @Gedmo\Timestampable(on="create")
+     */
+    private $postedAt;
+
+    /**
+     * @ORM\Column(length=50, nullable=true)
+     */
+    private $firstName;
+
+    /**
+     * @ORM\Column(length=50, nullable=true)
+     */
+    private $lastName;
+
+    /**
+     * @ORM\Column(nullable=true)
+     */
+    private $phone;
+
+    /**
+     * @ORM\Column(nullable=true)
+     *
+     * @Assert\Email
+     * @Assert\Length(max=255, maxMessage="common.email.max_length")
+     */
+    private $emailAddress;
+
+    /**
+     * @ORM\Column(type="boolean")
+     */
+    private $agreedToStayInContact = false;
+
+    /**
+     * @ORM\Column(type="boolean")
+     */
+    private $agreedToJoinParisOperation = false;
+
+    /**
+     * @var DataAnswer[]|Collection
+     *
+     * @ORM\OneToMany(targetEntity="DataAnswer", mappedBy="dataSurvey", cascade={"persist", "remove"})
+     *
+     * @Assert\Valid
+     */
+    private $answers;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="Survey")
+     * @ORM\JoinColumn(onDelete="CASCADE", nullable=false)
+     */
+    private $survey;
+
+    public function __construct(Survey $survey = null, string $firstName = null, string $lastName = null)
+    {
+        $this->survey = $survey;
+        $this->firstName = $firstName;
+        $this->lastName = $lastName;
+        $this->answers = new ArrayCollection();
+    }
+
+    public function getId(): ?int
+    {
+        return $this->id;
+    }
+
+    public function setAuthor(Adherent $author): void
+    {
+        $this->author = $author;
+    }
+
+    public function getAuthor(): ?Adherent
+    {
+        return $this->author;
+    }
+
+    public function getPostedAt(): ?\DateTime
+    {
+        return $this->postedAt;
+    }
+
+    public function setPostedAt(\DateTime $postedAt): void
```

Comment:
I think we can remove this setter as this field is entirely managed by Doctrine extension.